pkg/util/parameters.go

```diff
@@ -0,0 +1,180 @@
+package util
```

Comment:
This file needs an Amazon license header

Reply:
Done